src/generator/www/js/main.js

```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
+
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
+
+  $('#btn').click(function() { 
+
+  var form = document.querySelector('form');
+  var formData = new FormData(form);
+ 
+  formData.append( 'sessionfile', $( '#sessionfile' )[0].files[0] );
+  formData.append( 'speakerfile', $( '#speakerfile' )[0].files[0] );
+  formData.append( 'trackfile', $( '#trackfile' )[0].files[0] );
+  formData.append( 'locationfile', $( '#locationfile' )[0].files[0] );
+
+  var obj={
+    name:$name.val(),
+    email:$email.val(),
+    url :$url.val(),
+    files:formData
+    
+  }
```

Comment:
@aayusharora So basically in the POST request there will be a multi-part form
where i'll get name, email and url in post params ? 
And how'll I retrieve the files ?